config/webpack/webpack.config.base.js

```diff
@@ -124,13 +122,20 @@ module.exports = (env) => {
     plugins: [
       DEFINE_PLUGIN,
       BANNER_PLUGIN,
-      IGNORE_MOMENT_LOCALES,
     ],
     resolve: {
+      alias: {
+        // NOTE: rjsf still depends on core-js@2, should be able to remove with rjsf v3
+        // core-js-pure is the core-js@3 equivalent of core-js/library from core-js@2
+        // https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md
+        'core-js/library/fn/array/fill': path.resolve(NODE, 'core-js-pure/features/array/fill'),
+        'core-js/library/fn/array/includes': path.resolve(NODE, 'core-js-pure/features/array/includes'),
+      },
       extensions: ['.js', '.css'],
+      fallback: { util: false },
```

Comment:
I think you can remove this if you upgrade to the upcoming `lattice-auth` release.